autorest-schemas/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@autorest/schemas",
+  "version": "1.0.0",
+  "description": "Internal package storing schemas",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
Does this need the `files` property to include those json files in the package?

Reply:
nope they get included with bundledDependencies. I think npm will include everything expect what's defined in npmignore